setup.py

```diff
@@ -68,7 +68,7 @@ def readme() -> str:
         packages=find_packages("./src"),
         zip_safe=False,
         entry_points={"pytest11": ["syrupy = syrupy"]},
-        extras_require={"dev": dev_requires},
+        extras_require={"dev": dev_requires, "test": test_requires},
```

Comment:
I remember this was removed before, just not sure why now

Reply:
We had no need for it I think.